### ISSUE TITLE
Accessibility toggle button has no state

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#1172]
The Turn Accessibility On button doesn't have a state to the screen reader as it uses button text and an aria label.

The issue is on the Screen Reader, if you have one you will notice that the state of the button (is it on or off) is not reflected (is not read by SR).

**Comment by moloko** [comment]
https://github.com/adaptlearning/adapt_framework/commit/a78fcf32e0c488e8ecae376d47fd8d1b677959de
